ToDoList/Services/TodoService.cs

```diff
@@ -57,4 +58,38 @@ public async Task<List<Todo>> Get()
 
         return todos;
     }
+
+    public async Task<Todo> Update(UpdateTodoDTO input)
+    {
+        var todo = await context.Todos.FirstOrDefaultAsync(x =>
+            x.Id == input.Id);
+
+        if (todo is null)
+        {
+            return null;
+        }
+
+        todo.Description = input.Description;
+        todo.Id = input.Id;
+        todo.Title = input.Title;
```

Comment:
Why don't using AutoMapper ?

Reply:
1. Not sure if the author wants Automapper 
2. The properties are not much so we can still do normal mapping here.